dev-python/libcloud/libcloud-3.1.0.ebuild

```diff
@@ -0,0 +1,49 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python3_{6,7,8} )
```

Comment:
No 3.9?

Reply:
requests-mock and lockfile first ;) Upstream says they support 3.65->3.8, but does not say about 3.9

Reply:
Lockfile is deprecated, but still used in tests. I filed an issue upstream. In the meantime I'm making a PR for it as deprecated does not mean dead I guess.

Reply:
I'll test it all but will probably make one PR with one commit for missing test dep for libcloud, one for lockfile, one for requests-mock, and one for libcloud vbump+port to py3.9.

Reply:
As long as tests pass on 3.9 and there's no reported bugs upstream on 3.9, we should be fine.